flink-java/src/main/java/org/apache/flink/api/java/DataSet.java

```diff
@@ -181,7 +181,17 @@ protected void fillInType(TypeInformation<T> typeInfo) {
 		return this.type;
 	}
 
+	/**
+     *  1. Check if the function is implemented by a scala object. Checks only if scala object function forbidden
+     *    is not disabled in the [[org.apache.flink.api.common.ExecutionConfig]]
+     *
+     *  2. Returns a "closure-cleaned" version of the given function. Cleans only if closure cleaning
+     *     is not disabled in the [[org.apache.flink.api.common.ExecutionConfig]]
+     */
 	public <F> F clean(F f) {
```

Comment:
Thanks for review,I tried to add a new API to DataSet, but I got a compile error, I think some maven plugin prevent me to change the API, could you tell me how to be able to pass the compile and API check?